src/components/styles/StyleLogOut.js

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+
+export const PLogOut = styled.p`
```

Comment:
It would be better to have all the styled-components in separated files (one for each styled-component).

Reply:
+1